### ISSUE TITLE
Output lines broken when printing multiline string in task (windows)

### ISSUE THREAD

**Issue by rbaradari** [gradle/gradle#1566]
When printing a multiline string during task execution the output seems to be shifted and looks broken up in the console. If the console is explicitly set to plain the output is correctly aligned.

It does not matter if the output is printed by `println` or `logger.lifecycle`. 

See below for example _build.gradle_ to reproduce.

This is related to #1546 but I assess the cause to be different when looking at the fixing PR.

### Expected Behavior
```
>gradle outputText --console plain
:outputText
###
# This is the first long line!
# This is the second long line!
# This is the third long line!
# This is the fourth long line!
# This is the fifth long line!
###


BUILD SUCCESSFUL
```

### Current Behavior
```
>gradle outputText
:outputText
###onnecting to Daemon
> I# This is the first long line!
> IDLE                           # This is the second long line!
> IDLE                                                          # This is the third long line!
                                                                                              # This is the fourth long line!
                                                                                                                             # This is the fifth long line!
     ###


BUILD SUCCESSFUL
```

### Context
The use case is a _help_ task that prints documentation and additional usage descriptions for the build.

### Steps to Reproduce (for bugs)
_build.gradle_:
```
task outputText {
        doLast {
        println """\
###
# This is the first long line!
# This is the second long line!
# This is the third long line!
# This is the fourth long line!
# This is the fifth long line!
###
"""
    }
}
```

### Your Environment
Nightly build (3.6-20170313000015+0000)
Windows 10 Pro Command Prompt

**Comment by lacasseio** [comment]
Thanks @rbaradari for reporting this issue. We will look into fixing this soon.

**Comment by lacasseio** [comment]
This will be fixed in Gradle 3.5 RC1. Thanks for reporting this @rbaradari.
